.github/workflows/docker.yml

```diff
@@ -0,0 +1,65 @@
+name: Docker
+
+permissions:
+  packages: write
+
+on:
+  push:
+    branches:
+      - '**'
+    tags:
+      - 'v*.*.*'
+  pull_request:
+
+jobs:
+  main:
+    runs-on: ubuntu-latest
+    steps:
+      -
+        name: Checkout
+        uses: actions/checkout@v2
+      -
+        name: Docker meta
+        id: docker_meta
+        uses: docker/metadata-action@v3
+        with:
+          images: ghcr.io/${{ github.repository_owner }}/agora
+          tags: |
+            type=ref,event=branch
+            type=ref,event=pr
+            type=semver,pattern=v{{version}}
+            type=semver,pattern=v{{major}}.{{minor}}
+            type=semver,pattern=v{{major}}
+            type=sha
+      -
+        name: Set up QEMU
```

Comment:
Is QEMU necessary if it builds for Linux? I have no idea how this stuff works.

Reply:
QEMU allows for multi-platform build. Raspberry Pi uses arm64, so we need it.